application/forms/PreferenceForm.php

```diff
@@ -185,7 +186,15 @@ public function createElements(array $formData)
                 false
             );
         }
-
+        $config = Config::app()->getSection('global');
+        if (!isset($config->config_resource)) {
```

Comment:
```suggestion
        if (! isset($config->config_resource)) {
```
There should be a space after `!` operator.

Reply:
Really? Is this always the case? I've seen it without spaces elsewhere.